locales/en/messages.json

```diff
@@ -52,6 +52,13 @@
     "portsSelectManual": {
         "message": "Manual Selection"
     },
+    "portsSelectVirtual": {
+        "message": "Virtual Mode (Experimental)",
+        "description": "Enables a virtual FC to let navigate by the Configurator without the need of an FC"
```

Comment:
Maybe `Configure a Virtual Flight Controller without the need of a physical FC.`